README.md

```diff
@@ -0,0 +1,40 @@
+# EOWeb Client
+
+Web-based Endless Online client built with TypeScript.
+
+## Quick Start
+
+1. **Clone and install**:
+   ```bash
+   git clone https://github.com/sorokya/eoweb.git
+   cd eoweb
+   npm install
+   ```
+
+2. **Extract game graphics** (Required):
+   - Use [extract-egf-images](https://github.com/ExileStudios/extract-egf-images) to extract from `.egf` files
+   - Place extracted images in `public/gfx/`
```

Comment:
Extracted image directories should go in `public/` (e.g `public/gfx001/101.png`)